leistungsbeschreibung/README.md

```diff
@@ -1,57 +1,57 @@
 
+Der durch die CashboxId identifizierte Konfigurationscontainer (Cashbox) kann in verschiedene Plattformen und Betriebssysteme integriert werden. Die Erstellung und Verwaltung der Konfiguration inklusive der dazugehörigen Komponenten erfolgt im marktbezogenen fiskaltrust.Portal. Der fiskaltrust.SecurityMechanism wird von der Queue-Komponente und der SCU-Komponente (Signaturerstellungseinheit) bereitgestellt. Die SCU-Komponente stellt die Anbindung an den marktbezogenen Sicherheitsmechanismus (z.B. TSE) dar.
 
 
 # Revisionssichere Daten-as-a-Service
 
-Kassenbetreiber erhalten revisionssichere Daten-as-a-Service, indem sie ein Pos-System verwenden, das die fiskaltrust.Middleware enthält. 
+Kassenbetreiber erhalten revisionssichere Daten-as-a-Service, indem sie ein Kassensystem verwenden, das die fiskaltrust.Middleware verwendet. 
 
-## fiskaltrust-cloud
+## fiskaltrust.Cloud
 
-Die fiskaltrust cloud befindet sich in einer tiefen Integration mit der fiskaltrust-Middleware, die die Daten als Teil des Sicherheitsmechanismus in die cloud spiegelt. Die Datenintegrität wird durch den Hash-Wert der Requests und Responses sichergestellt, und die Vollständigkeit kann durch Nachverfolgung der Kassenbeleg-Hash-Kette überprüft werden. Dies bedeutet, dass alle Originaldatensätze erhalten bleiben und unveränderlich sind. Der Schutz vor Datenverlust wird durch eine Lösung mit standortverteilten Daten-Speichern gewährleistet. Die Daten werden – entsprechend den steuerlichen Bestimmungen – in deutschen Rechenzentren gespeichert.
+Die fiskaltrust.Middleware, spiegelt die Daten als Teil des fiskaltrust.SecurityMechanism in die fiskaltrust.Cloud. Die Datenintegrität wird durch den Hash-Wert der Requests und Responses sichergestellt, und die Vollständigkeit kann durch Nachverfolgung der Hash-Kette überprüft werden (s.o.). Dies stellt sicher, dass alle Originaldatensätze erhalten bleiben und nicht verändert werden. Der Schutz vor Datenverlust wird durch unsere Lösung mit standortverteilten Daten-Speicherung gewährleistet. Die Daten werden in deutschen Rechenzentren gespeichert.
 
 ## POS Archiv
 
-Das POS Archiv speichert Daten, basierend auf die übermittelten Belege über die fiskaltrust.Middleware, über einen Zeitraum von 10 Jahren. Zu diesem Zeitpunkt wird auch das spezielle Journal der erstellten unterschriebenen Geldeingänge abgedeckt.
+Das fiskaltrust POS Archiv speichert die von der fiskaltrust.Middleware übegebenen Belegdaten über einen Zeitraum von 10 Jahren. Über das fiskalttrust.Portal kann auf die Daten zugegriffen werden.
 
-Bei Auftreten einer monatlichen Quittung wird ein Datencheck zur Konsistenz durchgeführt und eine zusätzliche Exportdatei erstellt und auf einem unveränderlichen Speicher in der fiskaltrust-Cloud gespeichert.
+Bei Erstellen eines Monatsabschuss wird ein Datencheck zur Konsistenz der Daten durchgeführt und eine zusätzliche Exportdatei erstellt die ebenfalls in der fiskaltrust.Cloud gespeichert wird.
```

Comment:
"Beim Erstellen eines Monatsabschusses wird ein Check der Konsistenz der Daten durchgeführt und eine zusätzliche Exportdatei erstellt, die ebenfalls in der fiskaltrust.Cloud gespeichert wird."

Reply:
Danke fürs Feedback, ich habe es in meinem aktuellen Commit berücksichtigt: https://github.com/christian-rogobete/productdescription-de-doc/commit/2805ee1b9398e0f0b6436d027e64f58da528cd86